great_expectations/render/components.py

```diff
@@ -125,6 +125,7 @@ class LegacyDescriptiveRendererType(str, Enum):
     VALUE_COUNTS_BAR_CHART = ".".join([LegacyRendererType.DESCRIPTIVE, "value_counts_bar_chart"])
 
 
+@public_api
 class RenderedContent:
```

Comment:
Idk how much we care about renderer stuff being public but we already had some classes/methods in our docs so I added decorators to any violations

Reply:
@NathanFarmer What are your thoughts here? Should this be public or is this internal to our codebase?

Reply:
I don't think this is public any longer now that we have removed support for custom Expectations. Users won't be writing their own renderers with `UnexpectedRowsExpectation`.